### PR TITLE
Add some xCAT documentation writing guidelines

### PR DESCRIPTION
**Pull request by gurevichmark**
Additional documentation writing guidelines.

### CODE REVIEW

File: docs/source/developers/guides/docs/doc_guidelines.rst [hunk 1]
```diff
@@ -62,6 +62,24 @@ Enumerated List
...
 
+Include another file
+--------------------
+
+To add contents of a document file inside another file, use ``.. include::``. This is usefull when a common information needs to be displayed in multiple files, whithout the use of a hyperlink.
+::
+
+ .. include:: config_common.rst
+
+..
```
Comment (daniceexi) [inline docs/source/developers/guides/docs/doc_guidelines.rst]: What's this two ``..` for?

File: docs/source/developers/guides/docs/doc_guidelines.rst [hunk 2]
```diff
@@ -62,6 +62,24 @@ Enumerated List
...
+
+ ``Note:`` Do not put customized link targets, such as ``.. _my_link_taget:`` inside the file to be included. If you do, a warning for a duplicate label will be displayed during the documentation build process.
+
+Index file
+--------------------
```
Comment (daniceexi) [inline docs/source/developers/guides/docs/doc_guidelines.rst]: The `-------------` too long.

File: docs/source/developers/guides/docs/doc_guidelines.rst (hunk 1 above)
Comment (gurevichmark) [inline docs/source/developers/guides/docs/doc_guidelines.rst]: Removed, not needed.

File: docs/source/developers/guides/docs/doc_guidelines.rst (hunk 2 above)
Comment (gurevichmark) [inline docs/source/developers/guides/docs/doc_guidelines.rst]: Fixed under a new commit.

**Comment by daniceexi** [comment]
I have reviewed the change and agree to merge.

**Comment by gurevichmark** [comment]
@daniceexi I see you made the comment that you agree with the changes, but this pull request is still Open. Can you please merge it if there are no additional comments ?

**Pull request merged**